views/extension/_form.php

```diff
@@ -70,7 +70,8 @@
     </div>
     <div class="row">
         <div class="col-md-9">
-            <?= $form->field($model, 'description', ['options' => ['class' => 'nopackagist']])->textarea(['class' => 'markdown-editor']) ?>
+            <?= $form->field($model, 'description', ['options' => ['class' => 'nopackagist']])->textarea(['class' => 'markdown-editor', 'title' =>
+                'text']) ?>
```

Comment:
```suggestion
                'Description']) ?>
```

but... are you sure we need it?

Reply:
Yes, this is a small fix. It's good anyway